easybuild/easyconfigs/m/MIGRATE-N/MIGRATE-N-4.2.14-foss-2018a.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'ConfigureMake'
+
+name = 'MIGRATE-N'
+version = '4.2.14'
+
+homepage = 'http://popgen.sc.fsu.edu/Migrate/Migrate-n.html'
+description = """
+Migrate estimates population parameters, effective population sizes
+and migration rates of n populations, using genetic data.  It 
+uses a coalescent theory approach taking into account history of 
+mutations and uncertainty of the genealogy. 
+"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = [
+    'http://popgen.sc.fsu.edu/currentversions/',
+    'http://popgen.sc.fsu.edu/oldversions/%(version_major)s.x/%(version_major_minor)s',
+    'http://popgen.sc.fsu.edu/oldversions/',
+    'http://popgen.sc.fsu.edu/newversions/',
+]
+sources = ['migrate-%(version)s.src.tar.gz']
+patches = [
+    'migrate-%(version)s_install.patch',
+]
+checksums = [
+    'a4387dda154d5df6d8943b6edba44e9d29595f1a5853a5909f69c4caf7aed7c3',  # migrate-4.2.14.src.tar.gz
+    'b84eea6c8e76af3ef2a71aa566962cf6e49c3ec9c808993d85dc74e86e530c94',  # migrate-4.2.14_install.patch
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('glibc', '2.17'),
```

Comment:
@RvDijk How well did you test the result installation? You typically need to be very careful when including your own `glibc`... Is it an option to make this a build dep?

Reply:
@boegel I'll look into some testing now, probably build dep is preferred here.

Reply:
@boegel on this machine including glibc is breaking everything (also as a build dep) ... 
Probably and hopefully the `xlocale.h` inclusion isn't necessary, so I'll try a patch that will exclude it.